docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -8,253 +8,230 @@ videoBanner: UFhT52d5bYg
 
 ## Prerequisites
 
-Collect a few pieces of information about your Redshift cluster which you'll
-need to configure access.
+- Teleport version `(=teleport.version=)`.
+- AWS account with Redshift databases and permissions to create and attach IAM
+  policies.
 
-**Get cluster identifier from the `General information` panel:**
+## Step 1/5. Install Teleport
 
-<Figure align="center" bordered caption="Redshift cluster ID">
-![Redshift cluster ID](../../../img/database-access/guides/redshift/cluster-id@2x.png)
-</Figure>
-
-**Copy cluster endpoint from the `General information` panel:**
-
-<Figure align="center" bordered caption="Redshift cluster endpoint">
-![Redshift cluster endpoint](../../../img/database-access/guides/redshift/cluster-endpoint@2x.png)
-</Figure>
-
-You'll only need the `host:port` portion of the endpoint.
-
-**Take note of database user name and database name you will use to connect.**
-
-If you haven't created any database users, default database admin user name and
-maintenance database are displayed on the `Database configurations` panel of the
-`Properties` tab:
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
 
-<Figure align="center" bordered caption="Redshift cluster user/database">
-![Redshift cluster user/database](../../../img/database-access/guides/redshift/cluster-user-db@2x.png)
+## Step 2/5. Create Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 3/5. Configure IAM
+
+### Create IAM policy for Teleport
+
+Teleport needs AWS IAM permissions to be able to:
+
+- Discover and register Redshift databases.
+- Manage IAM user or IAM role policies.
+
+Go to the [Policies](https://console.aws.amazon.com/iamv2/home#/policies) page
+and create a managed IAM policy for the database agent.
+
```

Comment:
I think there needs to be an introductory sentence or two before the `Tabs` component indicating how a user will know which IAM resource to create.